_maps/map_files/Pahrump/Pahrump-Surface-2.dmm

```diff
@@ -29652,14 +30480,13 @@
 /turf/open/floor/plasteel/f13/vault_floor/floor/floorsolid,
 /area/f13/brotherhood/surface)
 "miV" = (
-/obj/structure/fence{
-	dir = 1
-	},
-/turf/open/indestructible/ground/outside/dirt{
-	dir = 4;
-	icon_state = "dirt"
+/obj/item/stack/sheet/mineral/wood/twenty,
+/obj/item/clothing/shoes/f13/military/fashion/cowboy_boots{
+	pixel_x = -15;
+	pixel_y = 25
 	},
-/area/f13/wasteland)
+/turf/open/indestructible/ground/outside/dirt,
```

Comment:
![image](https://user-images.githubusercontent.com/49745305/219883169-eadb683a-dfb7-43ed-be80-b763c8643f09.png)
![image](https://user-images.githubusercontent.com/49745305/219883176-f4a205ac-e565-429d-8ce8-636e987b171f.png)
![image](https://user-images.githubusercontent.com/49745305/219883183-48b587ae-597c-4a7c-85ea-5e0e86416b38.png)
![image](https://user-images.githubusercontent.com/49745305/219883187-bd218a04-9206-489d-8539-6e67aab16a8d.png)


Fix ugly

Reply:
got it, purging the ugly

Reply:
![image](https://user-images.githubusercontent.com/72716882/219894634-97884074-45f5-49ed-bfe1-af3d22f962ef.png)
![image](https://user-images.githubusercontent.com/72716882/219896708-91d2d9e4-8823-4f39-9cc7-4b3e35b42fc9.png)
![image](https://user-images.githubusercontent.com/72716882/219896832-a5f02d22-4070-4893-b9c9-fd4a8739b8da.png)
![image](https://user-images.githubusercontent.com/72716882/219897024-9b1890b3-cf8e-40fb-8f2a-bb8d3ffcbff2.png)